river/InputManager.zig

```diff
@@ -37,6 +37,7 @@ server: *Server,
 new_input: wl.Listener(*wlr.InputDevice) = wl.Listener(*wlr.InputDevice).init(handleNewInput),
 
 idle: *wlr.Idle,
+idle_inhibit_manager: *wlr.IdleInhibitManagerV1,
```

Comment:
There's no need to store a pointer to this as it's never used and will free itself when the `wl.Server` is destroyed.

Reply:
Alright. I moved the creation to `Server.zig` as I just saw that all the other wlroots interfaces that free themselves sit there.